### DISCUSSION TITLE
Chained 128x64px panels

### DISCUSSION THREAD

**Discussion by reto88** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#120] [Q&A]
Has someone working 128x64px panels which can chained?

**Comment by kingofmummy** [comment]
just have one 128x64 (P2.5) and it works flawlessly with this fantastic libary 
![2021_08_08_18_56_IMG_6289](https://user-images.githubusercontent.com/43493092/128653822-3666fbd6-792a-4464-9af9-e8cfcc91e224.JPG)

btw, hopefully someone make an example with this libary+web portal+tp2m.net
